hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/endpoint/HermesAPIOperations.java

```diff
@@ -21,24 +26,33 @@ public HermesAPIOperations(HermesEndpoints endpoints) {
         this.endpoints = endpoints;
     }
 
-    public Response createGroup(String group, String supportTeam) {
-        return endpoints.group().create(group().withGroupName(group).withSupportTeam(supportTeam).build());
-    }
+    public void createGroup(String group) {
+        if (endpoints.group().list().contains(group)) {
+            return;
+        }
+        assertThat(endpoints.group().create(Group.from(group)).getStatus()).isEqualTo(CREATED.getStatusCode());
 
-    public Response createGroup(String group) {
-        return endpoints.group().create(Group.from(group));
+        waitAtMost(Duration.ONE_MINUTE).until(() -> {
```

Comment:
te ONE_MINUTE -> nie możemy korzystać z naszej klaski Waiter? i tam to upchnąć i potem centralnie sterować czasem?